README.md

```diff
@@ -145,10 +145,37 @@ $ cat /tmp/20200710182129_ksvc_creation_time.csv
   ktest-109,ktest-1,13,13,2,0,0,0,0,15,2,0,2,15
   ktest-110,ktest-1,14,13,1,0,0,0,0,18,3,0,3,18
 ```
-**Sample Dashboard**
+
+### As a Knative developer, I want to have a kperf dashboard to visualize the measurement result
+
+A visualized result is automatically generated by kperf during the measurement step to make the measurement data to be intuitive, which is a static HTML file including a chart and a table.
+
 ![service_creation_duration measurement](docs/kperf_dashboard.png)
 
+As we will have tons of measurement metrics for a single test (e.g. `revision_ready`, `configuartion_ready`, `ingress_ready` and etc.), it’s important to pick some key measurement metrics to analyze the bottleneck of the Knative service creation lifecycle.
+
+The kperf dashboard is enabled to select or unselect the metrics to be observed, makes it much easier for developers to find out the most time consuming operations during the service creation. Besides, the kperf dashboard is supported to zoom in/out the data, developers can choose the full set or only a subset of the data to observe.
+
+The kperf dashboard is built based on an open source charting library [ECharts](https://echarts.apache.org/en/index.html), we are taking advantage of the rich features provided by this library, and also extend it's functionality by implementing some custom tools.
+
+Detailed description about the usage of the kperf dashboard:
+
+1. **Select Data Source** : choose another CSV file from local to be rendered in the page.
+
+2. **Value** : hover on the chart area to see detailed metric values under the same x axis.
+
+3. **Legend** : click on the legend to show/hide the metric line/bar in the chart. Metrics are auto-grouped by it's name prefix before the `_` symbol, e.g. in this sample chart, metrics `ingress_ready` `ingress_config_ready` and `ingress_lb_ready` are grouped together and assigned with a group icon ✤.
+
+4. Toolbox **Select/Unselect All** : click on 'select/unselect all' icon of the toolbox to show/hide all of the metric lines/bars in the chart.
+
+5. Toolbox **Select Group** : click on any 'group' icon of the toolbox to show the metric lines/bars under the target group in the chart and hide others.
+
+6. Toolbox **Switch Chart Type** : switch chart type between `line` and `bar`, `staked bar` and `unstacked bar`, `tiled line` and `untiled line`.
 
+7. Toolbox **Data Zoom** : enable to select a specific area to be zoomed in/out.
 
+8. Toolbox **Restore** : restore to initial chart configuration after switching chart type, data zooming and etc.
 
+9. Toolbox **Data View** : display raw data in current chart and update chart after being edited.
 
+10. Toolbox **Save as Image** : save current chart view as an image.
```

Comment:
Cool stuff!!! The sample chart and usage looks great! maybe a minor comment is icon 10 looks a little bit different from others.

Reply:
The numbers on the picture are special symbols, like ①②...⑧⑨⑩, the `10` just looks different from others :-(

Reply:
@zyjiaobj good to know, then never mind:-)